### PR TITLE
invoke-webrequest, iwr, windows/curl, windows/wget: add Indonesian tr…

### PR DESCRIPTION
**Pull request by reinhart1010**
…anslation

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

This is a continuation of #8177 by adding Indonesian translation to `Invoke-WebRequest`, `iwr`, `windows/curl`, and `windows/wget`.

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

File: pages.id/windows/invoke-webrequest.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Invoke-WebRequest
+
+> Performs a HTTP/HTTPS request to the Web.
+> This command can only be used through PowerShell.
+> More information: <https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/invoke-webrequest>.
```
Comment (marchersimon) [inline pages.id/windows/invoke-webrequest.md:5]: This sounds a little too English for may taste.

File: pages.id/windows/invoke-webrequest.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Unduh konten URL ke file:
+
+`Invoke-WebRequest {{http://example.com}} -OutFile {{jalan/menuju/file}}`
+
+- Send form-encoded data (POST request of type `application/x-www-form-urlencoded`):
```
Comment (marchersimon) [inline pages.id/windows/invoke-webrequest.md:11]: Same here.

**Comment by navarroaxel** [comment]
@reinhart1010 do you want to reopen and finish this work?

**Comment by reinhart1010** [comment]
Yes

**Pull request closed without merging**